lib/cubits/user/user_cubit.dart

```diff
@@ -19,33 +21,104 @@ class UserCubit extends Cubit<UserState> {
     final either = await _accountRepository.getUser();
 
     if (either.isRight) {
-      var user = either.right;
+      final either2 = await _enrichWithProgramme(either.right);
```

Comment:
not really sure what to do with these. The programme changes needs to be applied each time we update a user, we should find a way to avoid this. Any ideas?